colourblind8/colourblind8.py

```diff
@@ -40,13 +179,14 @@ def plot_scatter(self, x, y, alpha = 1,  labels = None, palette = None,
                           color = colours[idx],
                           alpha = alpha,
                           label = labels[idx])
+			ax.legend(title = legend_title, bbox_to_anchor=(1.01, 1))
         else:
             for idx, values in enumerate(y):
                 ax.scatter(x,
                            values,
                            color = colours[idx],
                            alpha = alpha)
-
+		# if the following parameters are entered, Colourblind8 will apply the theme
```

Comment:
I'm not sure this comment is neccessary.

Reply:
I think it is necessary for viewers to understand the whole flow of our package.